### ISSUE TITLE
Campaigns get sent to junk

### ISSUE THREAD

**Issue by Prowler3454** [gophish/gophish#668]
When an email is tested is successfully goes through in Microsoft outlook but when the campaign is launched that very same email gets sent straight to the junk mailbox. Strangely enough I have tried using emails that are pure text and they get sent straight to junk when tested. Any idea why this could be happening?

**Comment by jordan-wright** [comment]
I'm afraid there are countless reasons why an email could be sent to spam. However, I believe that if you look in the message source, you will see that there are differences in the messages. For example, the message in the campaign will have valid links pointing to your Gophish instance, whereas the test emails shouldn't have those links in place. This could be a key indicator that the email should be sent to spam.

**Comment by W1R3D-Code** [comment]
@Prowler3454, for an internal test, the best bet is always to whitelist your GoPhish IP / Campaign sender on your mail server.

If this isn't an option for whatever reason there is some basic config that is worth checking over to make sure you're giving them the best chance of getting through;

- Send from your own domain (or subdomain) rather than spoofing i.e. account.microsoft.com.somedomain.obscureTLD
- Get your ISP to setup a PTR record for your IP (you can test it here: https://mxtoolbox.com/ReverseLookup.aspx)
- Add SPF and DKIM DNS record, and encrypt your outgoing mail (i.e. https://www.linode.com/docs/email/postfix/configure-spf-and-dkim-in-postfix-on-debian-8)
- Setup a mx / mail / smtp record for the sending domain (some spam filters laboriously lookup those records and cross reference with the sending IP)

From a content perspective, there are lots of different and every evolving algorithms at play from different providers; but its largely accepted that there are basic guidelines you can follow to reduce your spam weighting;

- Don't use large images
- Don't use spammy language like 'urgent, act fast' 'fantastic deal'
- Make sure your HTML doesn't have any broken tags; just because chrome renders it ok, doesn't mean its to spec. Browsers do loads of clever things to fix broken HTML on the fly to mend the UX.
- Include a text version of your content

This a far from comprehensive list, someone might be able to add more :)

If you're still having issues, have a look at http://www.allaboutspam.com/email-server-test/

**Comment by Kaztar** [comment]
We have encountered a similar issue.  It is very specific though and doesn't have to do with the content.  When setting up the email, We have set it up two ways; one doesn't go to Junk, the other does. We have only tested this with Office 365.
[Email]@Gmail.com - Works (doesn't go to Junk),
[Visible Name] <[Email]@Gmail.com> - Automatically goes to the Junk folder.

We have resolved this by White listing the specific email. 
Hope this helps, resolves your issue.  Found this solution through: [Tips on keeping emails out of junk/spam folders? #628](https://github.com/gophish/gophish/issues/628)
Kaz
